tests/comfy_cli/test_tracking.py

```
@@ -0,0 +1,97 @@
+_ConfigManagerCls = ConfigManager.__closure__[0].cell_contents
+
+
+@pytest.fixture
+def tracking_module(tmp_path):
+    """Yield comfy_cli.tracking with a fresh tmp-path ConfigManager and a mocked Mixpanel client."""
+    config_dir = tmp_path / "comfy-cli"
+    config_dir.mkdir()
+    with patch.object(_ConfigManagerCls, "get_config_path", return_value=str(config_dir)):
+        cfg = _ConfigManagerCls()
+
+    import comfy_cli.tracking as tracking_mod
```

Pylint (4.0.5)

[convention] Import outside toplevel (comfy_cli.tracking)

(C0415)